fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/entities/RequestInfoEntity.java

```diff
@@ -2,22 +2,26 @@
 
 import lombok.Data;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 import java.time.LocalDate;
 
 @Data
 @Entity
 public class RequestInfoEntity {
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "request_info_generator")
+    @SequenceGenerator(name = "request_info_generator", sequenceName = "request_info_id_seq", allocationSize = 1)
```

Comment:
Using allocationSize 1 is a bad idea from a performance perspective as it is an entity-per-request class. Use 50 as sane default. See 
https://stackoverflow.com/questions/54741182/what-does-the-jpa-sequencegenerator-allocationsize-mean
Probably you will need to add `INCREMENTED BY` to liquibase sequence definition

Reply:
done